### PR TITLE
Create the storage checker module

### PR DESCRIPTION
**Pull request by poncovka**
Create a new DBus submodule for the storage checker.

Call the DBus method `SetConstraint` of the storage checker
module to set a constraint with a new value.

Refactorize the method names of the storage checker a little,
so only methods `add_contraint` and `set_constraint` could
change the constraints.

### CODE REVIEW

File: pyanaconda/modules/storage/checker/checker.py [hunk 1]
```diff
@@ -34,3 +35,14 @@ class StorageCheckerModule(KickstartBaseModule):
     def publish(self):
         """Publish the module."""
         DBus.publish_object(STORAGE_CHECKER.object_path, StorageCheckerInterface(self))
+
+    def set_constraint(self, name, value):
+        """Set a constraint to a new value.
+
+        KeyError will be raised if the constraint does not exist.
```
Comment (jkonecny12) [inline pyanaconda/modules/storage/checker/checker.py]: There is ``:raise:`` for this kind of documentation.

File: pyanaconda/modules/storage/checker/checker_interface.py [hunk 2]
```diff
@@ -27,4 +28,13 @@
 @dbus_interface(STORAGE_CHECKER.interface_name)
 class StorageCheckerInterface(InterfaceTemplate):
     """DBus interface for the storage checker module."""
-    pass
+
+    def SetConstraint(self, name: Str, value: Variant):
+        """Set a constraint to a new value.
+
+        KeyError will be raised if the constraint does not exist.
```
Comment (jkonecny12) [inline pyanaconda/modules/storage/checker/checker_interface.py]: The same here.

File: pyanaconda/modules/storage/checker/checker.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/modules/storage/checker/checker.py]: Fixed.

File: pyanaconda/modules/storage/checker/checker_interface.py (hunk 2 above)
Comment (poncovka) [reply pyanaconda/modules/storage/checker/checker_interface.py]: Fixed.

**Comment by poncovka** [comment]
Updated.

**Pull request merged**